src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  font-weight: 400px;
+  font-size: 10px;
+  color: #616070;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 3px 40px ;
+}
+
+.card__productName {
```

Comment:
```suggestion
.card__product-name {
```

Reply:
Be consistent about class names, in one case you're using camelCase, in other `-`